src/inputType/PanInput.ts

```diff
@@ -312,7 +312,7 @@ export class PanInput implements InputType {
     this._enabled = true;
     this._activeInput = activeInput;
     activeInput?.start.forEach((event) => {
```

Comment:
How about adding a logic that checks whether the `activeInput` is null, and throws an error if it is, for safety & clarity.